src/components/_navbar.scss

```diff
@@ -0,0 +1,128 @@
+// html, body {
+//   margin: 40px 0;
+//   padding: 0;
+// }
+
+.bar1, .bar2, .bar3 {
+  width: 2rem;
+  height: .3rem;
+  background-color: $primary;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+nav {
+  .collapsible input[id^=collapsible]:checked + button .bar1 {
+    -webkit-transform: rotate(-45deg) translate(-9px, 7px);
+    transform: rotate(-45deg) translate(-9px, 7px);
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar2 {
+    opacity: 0;
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar3 {
+    -webkit-transform: rotate(45deg) translate(-8px, -9px);
+    transform: rotate(45deg) translate(-8px, -9px);
+  }
+}
+
+nav {
+  display: flex;
+  z-index: 100;
+  width: 100%;
+  background-color: $main-background;
+  padding: .3rem;
+}
+nav.split-nav {
+  justify-content: space-between;
+}
+nav.fixed {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+}
```

Comment:
Different Classes for the same parent? How could you nest them together so you only have one `nav` with everything else nested inside?